x-pack/legacy/plugins/actions/index.ts

```diff
@@ -27,6 +27,13 @@ export function actions(kibana: any) {
       return Joi.object()
         .keys({
           enabled: Joi.boolean().default(false),
+          whitelistedHosts: Joi.alternatives()
```

Comment:
I think alternatives was here when the property was `string | string[]`?  So, I think we can remove the `alternatives()` and `try()` wrappers, just make it a `Joi.array()`?  Or maybe there's some joi sneakiness I don't yet understand :-)

It clearly works now, so fine with shipping this way, create an issue / project card if it should be fixed later.

Reply:
Yeah, that's not needed anymore, I'll fix in a small PR next